modules/simulations/batch-managed/modulestack.yaml

```diff
@@ -25,7 +28,9 @@ Resources:
               - "s3:Get*"
               - "s3:List*"
             Effect: Allow
-            Resource: ["arn:aws:s3:::addf*", "arn:aws:s3:::cdk*"]
+            Resource:
```

Comment:
Can you confirm if you need S3 permissions?

Reply:
I downscoped it to `arn:aws:s3:::${DagBucketName}/${DagPath}/*`